packages/core/src/fhir-deduplication/shared.ts

```diff
@@ -3,6 +3,9 @@ import dayjs from "dayjs";
 import utc from "dayjs/plugin/utc";
 import _, { cloneDeep } from "lodash";
 import { v4 as uuidv4 } from "uuid";
+import { errorToString } from "@metriport/shared";
+import { capture } from "../util";
+import { out } from "../util";
```

Comment:
nit shouldnt these be imported together in one line? 
do you use option+shift+O to format imports?

Reply:
Good call - done.